### ISSUE TITLE
Feature Request: render html5 tags such as <a> <img> <video>

### ISSUE THREAD

**Issue by cailiang9** [quantopian/qgrid#104]
We want to build an annotation tool based on jupyter notebook.  qgrid supports editing which is very convenient. It would be better if it supports html tags. So it can visualize image/video content for annotators directly.

**Comment by TimShawver** [comment]
Would you expect the rows to get taller to be able to display images/videos? The reason I ask is one of the limitations of SlickGrid is that all of the rows have to be the same height.

**Comment by cailiang9** [comment]
Yes. But same height is not a big issue. And I don't know how to display image/video in qgrid.

**Comment by TimShawver** [comment]
OK thanks...it's probably not possible yet, I'll have to look into it next time I work on a new release. Just was wondering how you would expect it to work.
